ui/app/AppLayouts/Profile/panels/ProfileDescriptionPanel.qml

```diff
@@ -0,0 +1,96 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Controls.Validators 0.1
+
+import utils 1.0
+import "../controls"
+
+Item {
+    id: root
+
+    property alias displayName: displayNameInput
```

Comment:
Could you pls change it to more suitable name, cos `displayName` means `text` not component